api/src/main/java/org/openmrs/module/fgh/mpi/MpiHttpClient.java

```diff
@@ -62,32 +40,42 @@ public class MpiHttpClient {
 	 * @throws Exception
 	 */
 	public Map<String, Object> getPatient(String patientUuid) throws Exception {
+		MpiContext mpiContext = MpiContext.initIfNecessary();
+		
 		log.info("Looking up patient record from MPI with OpenMRS uuid: " + patientUuid);
 		
 		if (log.isDebugEnabled()) {
 			log.debug("Searching for patient from MPI with OpenMRS uuid: " + patientUuid);
 		}
 		
-		if (openmrsUuidSystem == null) {
-			synchronized (MpiHttpClient.class) {
-				openmrsUuidSystem = MpiUtils.getGlobalPropertyValue(GP_UUID_SYSTEM);
-			}
-		}
+		String query = REQ_PARAM_SOURCE_ID + "=" + mpiContext.getOpenmrsUuidSystem() + "|" + patientUuid;
+		
+		Map<String, Object> pixResponse = null;
+		
```

Comment:
This looks weird, why not combine the 2 lines i.e. 53 and 55 into a single variable declaration and assignment instead of using null at first